product_variant_available_in_pos/models/product_template.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2016 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ProductTemplate(models.Model):
+
+    _inherit = 'product.template'
+
+    def _update_available_in_pos(self, vals):
```

Comment:
Please, add an `ensure_one()` here, as the method is meant to be used only on a single record at time.